### PR TITLE
changes

### PR DESCRIPTION
**Pull request by dsl233**
Sorry guys, I am not familiar with github and cause a lot of problems. Now I upload the review parts.

### CODE REVIEW

File: server/views/review/review.handlebars [hunk 1]
```diff
@@ -0,0 +1,126 @@
+<body>
```
Comment (leojacob7) [inline server/views/review/review.handlebars:1]: can you remove the <body > its already there in main

File: server/views/review/review.handlebars [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    {{!-- <h1>Just test my dbfunctions!</h1>
+    <form action="/reviews/" method="POST" id="login-form">
+    <label for="review" style="color:rgb(205, 183, 247);">Review</label>
+    <input type="text" id="review" name="review" value="{{review}}">
+    <input type="text" id="rating" name="rating" value="{{rating}}">
+    <button type="submit" id="ok">Submit</button>
+    </form> --}}
```
Comment (leojacob7) [inline server/views/review/review.handlebars:8]: I think we should remove them ?

**Review by leojacob7 — CHANGES_REQUESTED** [review]

File: server/views/review/review.handlebars (hunk 1 above)
Comment (leojacob7) [reply server/views/review/review.handlebars:1]: I think we should check our page source in html validator.

Comment (dsl233) [reply server/views/review/review.handlebars]: Ok, I will correct them and push it again.

**Review by leojacob7 — APPROVED** [review]

**Pull request merged**